documentation/modules/auxiliary/admin/http/netgear_r6700_pass_reset.md

```diff
@@ -0,0 +1,114 @@
+## Description
+
+This module exploits a buffer overflow vulnerability in the upnpd daemon (/usr/sbin/upnpd), running on the router Netgear R6700v3, ARM Architecture, firmware version V1.0.4.82_10.0.57 and V1.0.4.84_10.0.58.
+The vulnerability can only be exploited by an attacker on the LAN side of the router, but the attacker does not need any authentication to abuse it. After exploitation, an attacker will be able to use a default admin password to login to web interface and use a 'telnetenable' module to gain root shell.
+
+This vulnerability was discovered and exploited at Pwn2Own Tokyo 2019 by the team Flashback (Pedro Ribeiro + Radek Domanski).
+
+## Vulnerable Application
+
+* Netgear R6700v3 firmware version V1.0.4.82_10.0.57
+* Netgear R6700v3 firmware version V1.0.4.84_10.0.58
+
+[Netgear R6700v3 Firmware V1.0.4.82_10.0.57](http://www.downloads.netgear.com/files/GDC/R6700v3/R6700v3-V1.0.4.84_10.0.58.zip)
```

Comment:
This download link looks like its for v1.0.4.84_10.0.58, not for v1.0.4.82_10.0.57. Can you confirm this is the right link?

Reply:
Sorry, I mixed up when linking that firmware, It's fixed now.